packages/waku/src/lib/utils/buffer.ts

```diff
@@ -0,0 +1,16 @@
+export const parseFormData = async (
+  buffer: ArrayBuffer,
+  contentType: string,
+): Promise<FormData> => {
+  const response = new Response(buffer, {
+    headers: {
+      'Content-Type': contentType,
+    },
+  });
+  return response.formData();
+};
+
+export const bufferToString = (buffer: ArrayBuffer): string => {
+  const enc = new TextDecoder('utf-8');
```

Comment:
Can use the default. https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder/TextDecoder

```suggestion
  const enc = new TextDecoder();
```